lib/gh-checks.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+const chalk = require('chalk');
+const execa = require('execa');
+const ghGot = require('gh-got');
+const hostedGitInfo = require('hosted-git-info');
+const pTimeout = require('p-timeout');
+const pWaitFor = require('p-wait-for');
+const terminalLink = require('terminal-link');
+const util = require('./util');
+
+const THIRTY_SECONDS = 1000 * 30;
+const THIRTY_MINUTES = 1000 * 60 * 30;
+
+module.exports = task => {
+	return execa.stdout('git', ['cherry'])
+		.then(stdout => {
```

Comment:
Can you convert this file to use async/await? [`np` now targets Node.js 8](https://github.com/sindresorhus/np/commit/223ba6bf1787e2d864a94c3ad53a1446cff6b482).

Reply:
Also bump `gh-got` to v8.

Reply:
Converted to async/await in a34507a9d170c00219ac4be7de7c7e115a230d76. Bumped `gh-got` in 944875f2c8265b413bc27263a2f18f002ba45fc8.